package.json

```diff
@@ -1,7 +1,6 @@
 {
   "dependencies": {
-    "serverless": "^1.19.0",
-    "serverless-cf-vars": "^0.1.0",
-    "serverless-prune-plugin": "^1.1.1"
```

Comment:
Removed serverless prune because we're not using versioned functions in serverless framework